### ISSUE TITLE
MNIST

### ISSUE THREAD

**Issue by muhammadzulfanazhari** [chesterkuo/imageclassify-movidius#1]
do you have any idea how to train mnist dataset?

**Comment by chesterkuo** [comment]
did you mean how to use tensorflow to train mnist data or ?

**Comment by muhammadzulfanazhari** [comment]
yes, but based on mobilenet

**Comment by chesterkuo** [comment]
yes, it could be. you need to create a dataset with tfrecords for mnist, check following example to get how to create tfrecords.

https://github.com/chesterkuo/imageclassify-movidius/tree/master/mobilenet/tf-record

**Comment by chesterkuo** [comment]
Another very good example to reuse mnist dataset in tensorflow.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/

**Comment by muhammadzulfanazhari** [comment]
thank you, I'll try it
Is the mobilenet that you created for imagenet?
